app/controllers/utils/ControllerUtils.scala

```diff
@@ -61,16 +62,18 @@ class ControllerUtils @Inject()(configuration: Configuration) extends URIInforma
     formats: json.Reads[Map[String, String]]): Future[Result] =
     wsResponse.map { response =>
       response.status match {
-        case 200 => {
+        case OK => {
           if (response.body.contains("appStatusMessage")) {
             Logger.warn(
-              "GenerateResultBasedOnStatus Response Failed status:" + response.status + " json:" + " body:" + response.body)
-
+              s"[ControllerUtils][generateResultBasedOnStatus] GenerateResultBasedOnStatus Response Failed status: ${response.status}"
```

Comment:
Was there a reason why the json body was removed from the log?

Reply:
I could be wrong but i thought it was already being logged elsewhere

Reply:
Looks like its being logged down below in the case _ section. It might be useful to keep it as it holds the appStatusMessage and this looks like a generic method (unless im also wrong haha) but wait to see what others say.